From: Philip Withnall <withnall@endlessm.com>
Date: Wed, 29 Aug 2018 12:01:22 +0100
Subject: build: Drop unused HAVE_DLFCN_H definition
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

I can’t see this being used anywhere in GLib, or in my /usr/include
directory. I’m also not sure how configure.ac ends up defining it — it’s
certainly as a side-effect of something, and not deliberate.

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Bug: https://gitlab.gnome.org/GNOME/glib/issues/1313
Origin: upstream, 2.59.0, commit:e17be33a0eb78182d3e1658084c6b0f9f6ca5e10
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 64521bf..d583603 100644
--- a/meson.build
+++ b/meson.build
@@ -1884,7 +1884,6 @@ have_sh = find_program('sh', required : false).found() # For glib-gettextize
 
 # FIXME: defines in config.h that are not actually used anywhere
 # (we add them for now to minimise the diff)
-glib_conf.set('HAVE_DLFCN_H', 1)
 glib_conf.set('STDC_HEADERS', 1)
 glib_conf.set('SIZEOF___INT64', 8)
 
